### ISSUE TITLE
Bug with illustration at the getting started page

### ISSUE THREAD

**Issue by VladimirShitov** [scikit-hep/awkward#3009]
### Which documentation?

Documentation website

### What needs to be documented?

In the "How do I compute reductions or summary statistics" section of the [getting starting page](https://awkward-array.org/doc/main/getting-started/index.html) there is a corrupted illustration:
<img width="1081" alt="image" src="https://github.com/scikit-hep/awkward/assets/35199218/0443e331-f21c-4e03-b5a6-08bc28a14655">

Note the 3rd row in the image to the right. You might want to change this illustration: https://awkward-array.org/doc/main/_images/example-reduction-sum-only.svg

**Comment by jpivarski** [comment]
That's true. I keep fighting this because it shows up fine in some SVG viewers and not in others. I've redrawn the image for some tutorials to ensure that it's fixed, but then I lose track of which is the fixed version and which is the broken version because they are rendered the same way in the SVG viewers that I have.

I've self-assigned this issue; it will be a reminder that I need some permanent solution to this problem. (All copies of the bad SVG must be replaced, all in one go, so that I don't accidentally pick it up and use it again.)

**Comment by jpivarski** [comment]
#3648 will soon fix this, but I don't seem to be able to connect this issue with that PR, so to be sure that this issue gets closed, I'm closing it now.
